Mta 1478 mta web reviewing apps (#799)
windup/windup-documentation: docs/topics/mta-web-reviewing-apps.adoc

.Procedure

. In *Migration* view, click *Application inventory*.
. Select the application you want to review.
+
[NOTE]
====
Only one application can be reviewed at a time.
====

> . Click image:kebab.png[] (three dots) at the right end of the row, and select *Review* from the drop-down menu. The application *Review* parameters appear in the main pane.

Reviewer: @sbeskin-redhat - please correct me if I am wrong here but is this not the `Options` menu

Reviewer 2: @anarnold97 Yes, correct. It should be "the Options menu {kebab}".  
{kebab} is in the attributes - :kebab: image:kebab.png[title="Options menu"]

> . Click image:kebab.png[] (three dots) at the right end of the row, and select *Review* from the drop-down menu. The application *Review* parameters appear in the main pane.

Reviewer: ```suggestion
. Click image:kebab.png[Options menu] at the right end of the row, and select *Review* from the drop-down menu. The application *Review* parameters appear in the main pane.
```

> . Click image:kebab.png[] (three dots) at the right end of the row, and select *Review* from the drop-down menu. The application *Review* parameters appear in the main view.

Reviewer: Wrong format

+
image::mta-web-assessment-review.png[Application review]

. Click *Proposed action* and select the action.
. Click *Effort estimate* and set the level of effort required to perform the assessment with the selected questionnaire.

> . In the *Business criticality* field enter how critical the application is to the business.

Reviewer: ```suggestion

. In the *Business criticality* field, enter how critical the application is to the business.
```
Missing comma

> . In the *Work priority* field enter the application's priority.

Reviewer: ```suggestion
. In the *Work priority* field, enter the application's priority.
```
missing comma

.Procedure

. In *Migration* view, click *Application inventory*.
. Select the application you want to review.
+
[NOTE]
====
Only one application can be reviewed at a time.
====

. Click image:kebab.png[Options menu] at the right end of the row, and select *Review* from the drop-down menu. The application *Review* parameters appear in the main pane.
+
image::mta-web-assessment-review.png[Application review]

. Click *Proposed action* and select the action.
. Click *Effort estimate* and set the level of effort required to perform the assessment with the selected questionnaire.

> 

Reviewer 3: Hi @HagayVider1, looks like there is an redundant empty line here
Other than that, LGTM, thanks.
